### PR TITLE
Feat/dynamic balancer gas price

### PR DESCRIPTION
**Pull request by sdgoh**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**Description**:

- Add global config for balancer max swap setting. 
- Get quote/order price api call with max swap param per Balancer transaction
- Get estimate gas limit based-on max swap only when global config balancer_max_swaps changes.

Issue #2672

### CODE REVIEW

File: hummingbot/core/utils/eth_gas_station_lookup.py [hunk 1]
```diff
@@ -28,6 +30,50 @@ def get_gas_price(in_gwei: bool = True) -> Decimal:
     return gas_price if in_gwei else gas_price / Decimal("1e9")
 
 
+def get_gas_limit(connector_name: str) -> int:
+    gas_limit = None
+    Lookup = EthGasStationLookup.get_instance()
```
Comment (Nullably) [inline hummingbot/core/utils/eth_gas_station_lookup.py:35]: object instance name should start with lower case.

File: hummingbot/core/utils/eth_gas_station_lookup.py [hunk 2]
```diff
@@ -72,6 +120,23 @@ def refresh_time(self):
     def gas_price(self):
         return self._gas_prices[self.gas_level]
 
+    @property
+    def gas_limit(self):
```
Comment (Nullably) [inline hummingbot/core/utils/eth_gas_station_lookup.py:124]: if this is a dictionary, might be better to name it with plural, e.g. gas_limits

**Review by Nullably — APPROVED** [review]
I left a few comments on naming convention. The rest looks good to me.

**Comment by sdgoh** [comment]
naming convention updated in a different branch.

**Pull request merged**